catalog-info.yaml

```diff
@@ -150,81 +155,3 @@ spec:
           access_level: "READ_ONLY"
         ingestion-team: {}
         search-productivity-team: {}
-
----
-apiVersion: "backstage.io/v1alpha1"
-kind: "Resource"
-metadata:
-  name: "connectors-nightly-dockers"
-  description: "Connectors Service Nightly Dockers"
-  links:
-    - title: "connectors Nightly Buildkite Jobs"
-      url: "https://buildkite.com/elastic/connectors"
-spec:
-  type: "buildkite-pipeline"
-  owner: "group:ingestion-team"
-  system: "buildkite"
-  implementation:
-    apiVersion: "buildkite.elastic.dev/v1"
-    kind: "Pipeline"
-    metadata:
-      name: "connectors-nightly-dockers"
-      description: "Connectors Service Nightly Dockers"
-      links:
-        - title: "Connector Service Nightly Pipeline"
-          url: "https://buildkite.com/elastic/connectors-nightly-dockers"
-    spec:
-      pipeline_file: ".buildkite/nightly_docker.yml"
-      provider_settings:
-        trigger_mode: "none"
-      repository: "elastic/connectors"
-      schedules:
-        Daily 8_14:
-          branch: '8.14'
-          cronline: '@daily'
-          message: "Builds and pushes daily `8.14` docker images"
-        Daily 8_15:
```

Comment:
Is it safe to remove this on `main`? Will Backstage still be able to run these daily jobs for 8.14 and 8.15 as long as this exists in those branches?

Reply:
> Will Backstage still be able to run these daily jobs

I think you mean Buildkite here: I'm really not sure unfortunately. If we register this change in Backstage, those scheduled jobs could be deleted from Buildkite?

Reply:
yea my guess would have been 'removal from a yaml file in `main` == removal from buildkite'

Reply:
Ok, so I should just change these to not run main, but leave them _in_ main for now? And I can add a YAML comment to explain that these will eventually phase out.

Reply:
I think that sounds like the safest option, yep 👍